### PR TITLE
Styles recipient bar

### PR DESCRIPTION
**Pull request by birdy-**
Needed by https://github.com/alkemics/dashboard-flux/pull/486

### CODE REVIEW

File: src/_address-bar.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    height: auto;
+    padding-top: 0.5em 0.5em 0 0.5em;
+    padding-bottom: 0;
+    > .recipient-bar-tag {
+        display: flex;
+        color: #fff;
+        background: $alk-green;
+        padding: 0 5px;
```
Comment (adlenafane) [inline src/_address-bar.scss]: Maybe use the `$padding-base-horizontal` / `$padding-xs-horizontal` from bootstrap?

File: src/_address-bar.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.recipient-bar {
+    @extend .form-control;
+    display: flex;
```
Comment (adlenafane) [inline src/_address-bar.scss]: Does it fallback a minimum on old (shitty d:) browsers

**Comment by adlenafane** [comment]
Couple of comments otherwise LGTM

**Comment by SamuelRamond** [comment]
Please use PR naming convention

**Comment by SamuelRamond** [comment]
LGTM :+1:

**Pull request merged**